BoxContentSDK/BoxContentSDK/Requests/BOXFileRequest.m

```diff
@@ -241,4 +241,81 @@ - (BOOL)shouldPerformBackgroundOperation
     return (self.associateID.length > 0 && self.requestDirectoryPath.length > 0);
 }
 
+- (void)setRepresentationRequestOptions:(BOXRepresentationRequestOptions)representationOptions, ... NS_REQUIRES_NIL_TERMINATION
+{
+    NSMutableArray *arguments=[[NSMutableArray alloc]init];
+    BOXRepresentationRequestOptions eachObject;
+    va_list argumentList;
+    if (representationOptions) {
+        [arguments addObject: [NSNumber numberWithUnsignedInteger:representationOptions]];
+        va_start(argumentList, representationOptions);
+        while ((eachObject = va_arg(argumentList, BOXRepresentationRequestOptions))) {
+            if ([NSNumber numberWithUnsignedInteger:eachObject] > 0) {
+                [arguments addObject: [NSNumber numberWithUnsignedInteger:eachObject]];
+            }
+        }
+        va_end(argumentList);
+    }
+    
+    self.representationsRequested = [[NSMutableOrderedSet alloc] initWithArray:arguments];
+}
+
+- (NSString *)formatRepresentationRequestHeader
+{
+    // Only process valid API options.
+    if ([self.representationsRequested containsObject:@(BOXRepresentationRequestOriginal)]) {
```

Comment:
This should already be removed by the time the options are set

Reply:
I exposed formatRepresentationRequestHeader for testing. I added this check again to bullet proof reps processing. If a dev calls, I wanted to make sure this was not included in the x-reps-hints header.